apiary.apib

```diff
@@ -168,7 +180,7 @@ Polls is a simple API allowing consumers to view polls and vote in them.
         }
         
 
-## Questions Collection [/provinces-info?provinces=[1,2,3]&relative=true]
+## Questions Collection [/provinces-info?relative=true]
```

Comment:
why you removed the provinces=[...]? I think we need the option to get the response not for all provinces.

Reply:
When do we need it just for a few? I thougt this method is for getting all province names for the multiselect in the frontend. Therefor we would not need the selection of province-ids in the rest call

Reply:
I have deleted it, because i thougth we would just need it for the app and all provinces together. But i can change it and them merge it with the deictricts call